internal/config/config.go

```diff
@@ -18,11 +18,12 @@ const (
 	hcloudNetwork  = "HCLOUD_NETWORK"
 	hcloudDebug    = "HCLOUD_DEBUG"
 
-	robotEnabled           = "ROBOT_ENABLED"
-	robotUser              = "ROBOT_USER"
-	robotPassword          = "ROBOT_PASSWORD"
-	robotCacheTimeout      = "ROBOT_CACHE_TIMEOUT"
-	robotRateLimitWaitTime = "ROBOT_RATE_LIMIT_WAIT_TIME"
+	robotEnabled                   = "ROBOT_ENABLED"
+	robotUser                      = "ROBOT_USER"
+	robotPassword                  = "ROBOT_PASSWORD"
+	robotCacheTimeout              = "ROBOT_CACHE_TIMEOUT"
+	robotRateLimitWaitTime         = "ROBOT_RATE_LIMIT_WAIT_TIME"
+	robotDisableForwardInternalIPs = "ROBOT_DISABLE_FORWARD_INTERNAL_IPS"
```

Comment:
What do you think about using `ROBOT_FORWARD_INTERNAL_IPS` with a default of `true`? Double negation is always prone for errors.

Reply:
> So let's switch to ROBOT_FORWARD_INTERNAL_IPS?

I am happy with that. Eventually I think we should add something more sane than 100 env vars for configuration anyway. (Flags, Config File)